src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Private-DNS-Generic.json

```
@@ -61,11 +61,24 @@
                     "description": "The delay in evaluation of the policy. Review delay options at https://learn.microsoft.com/en-us/azure/governance/policy/concepts/effect-deploy-if-not-exists"
                 },
                 "defaultValue": "PT10M"
-            }
+            },
+            "location": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Location (Specify the Private Endpoint location)",
+                  "description": "Specify the Private Endpoint location",
+                  "strongType": "location"
+                },
+                "defaultValue": "uksouth"
```

```suggestion
                "defaultValue": "northeurope"
```

to match the region we use in all the DINE policies by default